README.rst

```diff
@@ -53,7 +53,7 @@ To install ARIA from source, download the source tarball from
 **Ubuntu/Debian** (tested on Ubuntu14.04, Ubuntu16.04)::
 
     apt-get install -y python-dev gcc libffi-dev libssl-dev
-    pip install aria[ssh]
+    pip install apache-ariatosca[ssh]
```

Comment:
Yeah, it seems like Github has been malfunctioning today..

I don't have permissions for the AppVeyor account, only Apache Infra does; Supposedly it's easiest to simply re-push (after "modifying" the commit, e.g. by empty rebase).
Since this is only a README change, though, we could simply merge despite the tests having failed. It's simply less pretty to see the red X :)

Up to you, let me know whether you'd like to push again or I should merge right now
Thanks :)